<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>319</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>319</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AppManager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/application_view_tile.png</normaloff>icons/application_view_tile.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0,0,0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lName">
        <property name="text">
         <string>Nazwa</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="6">
       <widget class="QLineEdit" name="leName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lPath">
        <property name="text">
         <string>Ścieżka</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="6">
       <widget class="QLineEdit" name="pathToExe">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="pbBrowseExe">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/folder_browse</normaloff>:/icon/folder_browse</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lWorkDir">
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="6">
       <widget class="QLineEdit" name="leWorkDir">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="pbBrowseWorkDir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/folder_browse</normaloff>:/icon/folder_browse</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbOutput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>zapisuj logi</string>
        </property>
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pbStartApp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>włącz</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/play</normaloff>:/icon/play</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pbStopApp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>wyłącz</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/pause</normaloff>:/icon/pause</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pbOpenWorkDir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>otwórz folder roboczy</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/folder</normaloff>:/icon/folder</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="pbOpenLogDir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>otwórz folder z logami</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/logs</normaloff>:/icon/logs</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="pbStartTerminal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>otwórz terminal w folderze roboczym</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/app</normaloff>:/icon/app</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="pbSaveInfo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>zapisz dane</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/save</normaloff>:/icon/save</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pbStopAll">
        <property name="toolTip">
         <string>zatrzymaj wszystkie</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/stop</normaloff>:/icon/stop</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pbAddApp">
            <property name="toolTip">
             <string>dodaj wpis</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/add</normaloff>:/icon/add</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbRemoveApp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>usuń wpis</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/delete</normaloff>:/icon/delete</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pbMoveUp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>przesuń wyżej</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/up</normaloff>:/icon/up</iconset>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pbMoveDown">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>przesuń niżej</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/down</normaloff>:/icon/down</iconset>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="pbSettings">
            <property name="toolTip">
             <string>ustawienia</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/wrench</normaloff>:/icon/wrench</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pbDuplicateApp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>duplikuj wpis</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/duplicate</normaloff>:/icon/duplicate</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lwApps"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
